src/main/resources/static/js/gw.history.js

```diff
@@ -215,50 +215,74 @@ GW.history = {
 		"      <th scope=\"col\">Notes (Click to Edit)</th> "+
 		"      <th scope=\"col\">Status</th> "+
 		"      <th scope=\"col\">Action</th> "+
+        "      <th scope=\"col\">Checkpoint</th> "+
```

Comment:
I think I mentioned. This column should be removed.

Add a new button `restore` to the `Action` column.

Reply:
this line of code show up beside the action button like this:
<img width="1132" alt="image" src="https://github.com/ESIPFed/Geoweaver/assets/45752727/d8cbf026-0eb4-475d-9eda-b82a4f463459">